ConsoleApp3/Math.Services/LargestNumber.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Linq;
+
+namespace Math.Services
+{
+    public class LargestNumber
+    {
+        public static int? MaxFromArray(int[] array)
+        {            
+            int? result = null;
+
+            if (array == null || array.Length == 0)
+            {
+                return result;
+            }
+            result = array[0];
+            foreach (int currentValue in array)
+            {                
+                if (currentValue > result)
+                {
+                    result = currentValue;
+                }                
+            }
+            return result;
+        }
+        public static string MaxFromArray(string[] array)
+        {
+            string result;
+            if (array == null || array.Length == 0)
+            {
+                return result = "";
+            }
+            int[] numbersArray = Array.ConvertAll(array, int.Parse);
+           return result = MaxFromArray(numbersArray).ToString();            
```

Comment:
@rafaello-rm  w tej linijce zmienna `result` jest niepotrzebna.
Może to być zapisane tak:

 `return MaxFromArray(numbersArray).ToString();`